### DISCUSSION TITLE
Stable entitlement support

### DISCUSSION THREAD

**Discussion by PogoDigitalism** [Rapptz/discord.py#9692] [General] [answered]
Hi there, are  `entitlements` (as part of the interaction payload) supported yet in discord.py? I see that the current main version has support for Entitlements. I also did read that it will be an official feature of 2.4. However, the latest merge request on `entitlements` DID get merged and was said to be fully supported and operational. 
Does this mean I could clone the current main branch and include `entitlements` in my project? 

Cheers ❤️

**Comment by codeofandrin** [comment, marked as answer]
Yes it's on the main branch. You can always use the main branch to have the latest updates. However it's always recommended to use a stable/released version.

**Reply by PogoDigitalism** [reply]
Great, thanks. Any idea on when 2.4 will release as stable?

**Reply by codeofandrin** [reply]
No, only Danny (maintainer) can say that. I don't think it's planned because there are a few things Discord has to fix (afaik).

**Reply by PogoDigitalism** [reply]
What things are there to fix on discord's end? Is it regarding entitlements?

**Reply by codeofandrin** [reply]
I don't know exactly but I think it's about the premium subscription feature.
